workers/docker-worker/src/lib/features/artifacts.js

```diff
@@ -143,7 +154,7 @@ class Artifacts {
         let start = process.hrtime();
 
         let {digest, size} = await uploadToS3(taskHandler.queue, taskId, runId, stream,
-          entryName, expiry, headers);
+          entryName, expiry, headers, null, null, compress);
```

Comment:
I wasn't sure if there was a more idiomatic way of passing in the compress flag without these extra null parameters.